backend/api/Controllers/AreaController.cs

```diff
@@ -0,0 +1,302 @@
+        [ProducesResponseType(StatusCodes.Status400BadRequest)]
+        [ProducesResponseType(StatusCodes.Status409Conflict)]
+        [ProducesResponseType(StatusCodes.Status401Unauthorized)]
+        [ProducesResponseType(StatusCodes.Status403Forbidden)]
+        [ProducesResponseType(StatusCodes.Status500InternalServerError)]
+        public async Task<ActionResult<AreaResponse>> Create([FromBody] CreateAreaQuery area)
+        {
+            _logger.LogInformation("Creating new area");
+            try
+            {
+                var existingArea = await _areaService.ReadByInstallationAndName(area.InstallationCode, area.AreaName);
+                if (existingArea != null)
+                {
+                    _logger.LogWarning("An area for given name and installation already exists");
+                    return Conflict($"Area already exists");
+                }
+
+                var newArea = await _areaService.Create(area);
+                _logger.LogInformation(
+                    "Succesfully created new area with id '{areaId}'",
+                    newArea.Id
+                );
+                var response = new AreaResponse
+                {
+                    Id = newArea.Id,
+                    DeckName = newArea.Deck.Name,
+                    PlantCode = newArea.Plant.PlantCode,
+                    InstallationCode = newArea.Installation.InstallationCode,
+                    AreaName = newArea.Name,
+                    MapMetadata = newArea.MapMetadata,
+                    DefaultLocalizationPose = newArea.DefaultLocalizationPose,
+                    SafePositions = newArea.SafePositions
+                };
+                return CreatedAtAction(
+                    nameof(GetAreaById),
+                    new { id = newArea.Id },
+                    response
+                );
+            }
+            catch (Exception e)
+            {
+                _logger.LogError(e, "Error while creating new area");
+                throw;
+            }
+        }
+
+        /// <summary>
+        /// Add safe position to an area
+        /// </summary>
+        /// <remarks>
+        /// <para> This query adds a new safe position to the database </para>
+        /// </remarks>
+        [HttpPost]
+        [Authorize(Roles = Role.Admin)]
+        [Route("{installationCode}/{plantCode}/{deckName}/{areaName}/safe-position")]
+        [ProducesResponseType(typeof(AreaResponse), StatusCodes.Status201Created)]
+        [ProducesResponseType(StatusCodes.Status400BadRequest)]
+        [ProducesResponseType(StatusCodes.Status401Unauthorized)]
+        [ProducesResponseType(StatusCodes.Status403Forbidden)]
+        [ProducesResponseType(StatusCodes.Status500InternalServerError)]
+        public async Task<ActionResult<AreaResponse>> AddSafePosition(
+            [FromRoute] string installationCode,
+            [FromRoute] string plantCode,
```

Comment:
You don't really use plantCode and deckName except for the logger and the exception, do we really need them as inputs, then?